src/logging/logger.ts

```diff
@@ -112,38 +173,108 @@ export class Logger implements ZLUX.Logger {
     return configuredLevel >= level;
   };
 
+  private static createPrependingStrings(prependLevel?: boolean,
+                                         prependProcess?: boolean,
+                                         prependUser?: boolean): string[] {
+    let formatting = '';
+    if (prependProcess) {
+      formatting += Logger.processString;
+    }
+    if (prependUser && Logger.username) {
+      formatting += `${Logger.username} `;
+    } else if (prependUser && Logger.euid) {
+      formatting += `${Logger.euid} `;
```

Comment:
If there isnt username or euid, should just print "N/A" instead of omitting entirely as it does now.